courses/machine_learning/deepdive2/structured/labs/1a_explore_data_babyweight.ipynb

```diff
@@ -13,7 +13,7 @@
     "\n",
```

Comment:
<p>remove this heading, it's unnecessary.</p><p> </p><p>Also, I don't love having highlighetable hints. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>